racetrack_commons/racetrack_commons/deploy/image.py

```diff
@@ -1,27 +1,13 @@
 from racetrack_client.utils.url import join_paths
 
 
-def get_full_image(
-    docker_registry: str,
-    docker_registry_namespace: str,
-    image_type: str,
-    name: str,
-    tag: str,
-) -> str:
-    """Return full docker image name"""
-    return join_paths(docker_registry, docker_registry_namespace, image_type, f'{name}:{tag}')
-
-
-def get_fatman_base_image(docker_registry: str, registry_namespace: str, name: str, tag: str) -> str:
-    """Return full name of Fatman base image"""
-    return get_full_image(docker_registry, registry_namespace, 'fatman-base', name, tag)
-
-
-def get_fatman_image(docker_registry: str, registry_namespace: str, name: str, tag: str) -> str:
-    """Return full name of Fatman entrypoint image"""
-    return get_full_image(docker_registry, registry_namespace, 'fatman-entrypoint', name, tag)
-
-
-def get_fatman_user_module_image(docker_registry: str, registry_namespace: str, name: str, tag: str) -> str:
-    """Return full name of Fatman user-module image (from Dockerfile job type)"""
-    return get_full_image(docker_registry, registry_namespace, 'fatman-user-module', name, tag)
+def get_fatman_image(docker_registry: str, registry_namespace: str, name: str, tag: str, module_index: int = 0) -> str:
+    """
+    Return full name of Fatman image.
+    If a fatman is composed of multiple containers, the module number is included.
```

Comment:
The `module` parameter is not easily understandable in the context of images and containers.. just complaining, don't have good proposition here. Maybe more explanation what user should pass here, what are the expectations, what kind of logic it is modelling, would help. Btw, in the DockerBuilder::build you pass there `image_index` param.